docs/app/Examples/collections/Menu/Content/SubMenu.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Button, Icon, Input, Menu } from 'stardust'
+
+const {Item} = Menu
+
+const SubMenu = () => {
+  return (
+    <Menu vertical>
+      <Item>
+        <Input placeholder="Search..." />
+      </Item>
+
+      <Item>
+        Home
+
+        <Menu>
```

Comment:
Technically, it should be `Menu.Menu`.  Let me think about this one for a little.  I don't want to suddenly break the pattern of subcomponents matching SUI component part class names.